### PR TITLE
Add Azure to FTE exchange manager docs

### PR DESCRIPTION
**Pull request by sheajamba**
## Description
Adds Azure to the Exchange manager paragraph in the fault-tolerance execution docs.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]
Nice.

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst [hunk 1]
```diff
@@ -326,8 +326,10 @@ Exchange manager
 
 Exchange spooling is responsible for storing and managing spooled data for
 fault-tolerant execution. You can configure a filesystem-based exchange manager
-that stores spooled data in a specified location, such as an S3-compatible
-storage system, Google Cloud Storage (GCS), or a local filesystem.
+that stores spooled data in a specified location, such as :ref:`AWS S3
+<fte-exchange-aws-s3>` and S3-compatible systems, :ref:`Azure Blob Storage
+<fte-exchange-azure-blob>`, :ref:`Google Cloud Storage <fte-exchange-gcs>`, or a
```
Comment (arhimondr) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.rst:331]: I would remove the mention of `fte-exchange-local-filesystem`. This implementation is built only for local testing and is not designed for production use cases.

**Review by arhimondr — APPROVED** [review]
LGTM % comment

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/admin/fault-tolerant-execution.rst:331]: @sheajamba can you do that...

Comment (mosabua) [reply docs/src/main/sphinx/admin/fault-tolerant-execution.rst:331]: or add something like "for testing"

**Review by mosabua — APPROVED** [review]

**Pull request merged**